tests/data/synthetic experiment Header v231 no last time.ini

```diff
@@ -0,0 +1,24 @@
+[LOGIN]
+User = "Mr Test"
+Profile = "default"
+Software Version = "2.3.1"
+Experiment Date Time = "01/01/2020 12:34:56"
+
+[IMAGING MODES]
+Time Series = FALSE
+Motor Zstack = FALSE
+AOL Zstack = TRUE
+Functional Imaging = TRUE
+Volume Imaging = FALSE
+
+[FUNCTIONAL IMAGING]
+Functional Mode = "Patch"
+
+[VOLUME IMAGING]
+Functional Mode = "Point"
+
+[Intertrial FIFO Times]
+0.000000	0.000000
+1.000000	12.345678
+2.000000	12.567890
+3.000000
```

Comment:
I had assumed that in this case the whole line would be missing, not just the time, but maybe I'm wrong. I expect the tests would still pass even if the last line was completely removed?